instance-conf/httpd-2.4/bin/fixrootpath.pl

```diff
@@ -24,7 +24,7 @@
 #   --srcdir=path or string to be replaced 
 #   --dstdir=/path/to/destination/installation
 #
-# Pivotal Web Server patch distributions include two patterns to replace:
+# This package of Apache HTTP Server include two patterns to replace:
```

Comment:
"This package of Apache HTTP Server include"
should be
"This package of Apache HTTP Server includes"